diagnostic.md

```diff
@@ -21,13 +21,21 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function (event) {
+  event.preventDefault();
```

Comment:
this function doesn't need to take an event or prevent the default action, this does in the click registration